openstack/blockstorage/v3/volumetypes/doc.go

```diff
@@ -37,6 +37,14 @@ Example to create a Volume Type
 		panic(err)
 	}
 	fmt.Println(volumeType)
+
+Example to delete a Volume TYpe
+
+	typeID := "0fe36e73809d46aeae6705c39077b1b3"
```

Comment:
nit: this is supposed to be a UUID such as `ca33f1fa-3ff5-42d8-a8a8-4417ae2e9130`